src/cart/README.env

```diff
@@ -174,3 +174,15 @@ This file lists the environment variables used in CaRT.
  . CRT_TEST_CONT
    When set to 1, orterun does not automatically shut down other servers when
    one server is shutdown. Used in cart internal testing.
+
+ . D_CLIENT_METRICS_ENABLE
```

Comment:
Should these be added to the list in src/cart/crt_init.c as well?

Reply:
ah, sure.